app/controllers/BankDetailsController.scala

```diff
@@ -118,29 +119,41 @@ class BankDetailsController @Inject() (
           bankDetailsForm.isSoleSignatory.asBoolean
         )
 
-          def enterBankDetailsPageWithBarsError(error: FormError): Result = {
-            Ok(views.enterBankDetailsPage(
-              form.withError(error),
-              BankDetailsController.chooseTypeOfAccountUrl
-            ))
-          }
-
-        journeyService.updateDirectDebitDetails(request.journeyId, directDebitDetails)
-          .flatMap { _ =>
-            bankDetailsForm.isSoleSignatory match {
-              case IsSoleSignatoryFormValue.No =>
-                Future.successful(Redirect(routes.BankDetailsController.cannotSetupDirectDebitOnlinePage))
-
-              case IsSoleSignatoryFormValue.Yes =>
-                barsService.assessBankAccountReputation(directDebitDetails.bankDetails).map {
-                  case accountNumberIsWellFormattedNo() => enterBankDetailsPageWithBarsError(accountNumberNotWellFormatted)
-                  case sortCodeSupportsDirectDebitNo()  => enterBankDetailsPageWithBarsError(sortCodeDoesNotSupportsDirectDebit)
-                  case sortCodeIsPresentOnEiscdNo()     => enterBankDetailsPageWithBarsError(sortCodeNotPresentOnEiscd)
-                  // BARs check was successful
-                  case _                                => Redirect(routes.BankDetailsController.checkBankDetails)
-                }
+        // format: OFF
```

Comment:
does the formatter here really make it that ugly?

Reply:
no 🙂 it was just an experiment - happy to remove